ts/features/wallet/onboarding/cobadge/store/reducers/foundCoBadge.ts

```diff
@@ -37,7 +37,7 @@ const foundCoBadgeReducer = (
  * Remove from the list the disabled accounts
```

Comment:
Is this comment right? Can't understand where are filtered the disabled accounts